mysql-test/suite/rocksdb/t/use_direct_reads_writes.test

```diff
@@ -12,8 +18,8 @@
 --exec echo "wait" >$_expect_file_name
 shutdown_server 10;
 
-# Clear the log
---exec echo "" >$MYSQLTEST_VARDIR/log/mysqld.1.err
+# Add a delimiter to the log file
+--exec echo "[Note] use_direct_reads_write.test restart direct_reads" >> $MYSQLTEST_VARDIR/log/mysqld.$_server_id.err
```

Comment:
This will not work with MTR --repeat arg. A dedicated error log should be used instead (cheap to switch here as you are restarting anyway)

Reply:
You can see an example of a dedicated log in my pull request here https://github.com/facebook/mysql-5.6/pull/711/files#diff-61934317c60558ea9bedc26f897e9af1